src/widget/wslidercomposed.cpp

```diff
@@ -150,23 +204,88 @@ void WSliderComposed::paintEvent(QPaintEvent * /*unused*/) {
     QStylePainter p(this);
     p.drawPrimitive(QStyle::PE_Widget, option);
 
+    // Painter for the level bar
+//    QPainter pl(this);
+
     if (!m_pSlider.isNull() && !m_pSlider->isNull()) {
         m_pSlider->draw(rect(), &p);
     }
 
+    // Draw level bar underneath handle
+    if (m_dBarWidth > 0.0) {
+        drawBar(&p);
+    }
+
     if (!m_pHandle.isNull() && !m_pHandle->isNull()) {
         // Slider position rounded, verify this for HiDPI : bug 1479037
         double drawPos = round(m_handler.parameterToPosition(getControlParameterDisplay()));
+        QRectF targetRect;
         if (m_bHorizontal) {
             // The handle's draw mode determines whether it is stretched.
-            QRectF targetRect(drawPos, 0, m_dHandleLength, height());
-            m_pHandle->draw(targetRect, &p);
+            targetRect = QRectF(drawPos, 0, m_dHandleLength, height());
         } else {
             // The handle's draw mode determines whether it is stretched.
-            QRectF targetRect(0, drawPos, width(), m_dHandleLength);
-            m_pHandle->draw(targetRect, &p);
+            targetRect = QRectF(0, drawPos, width(), m_dHandleLength);
+        }
+        m_pHandle->draw(targetRect, &p);
+    }
+}
+
+void WSliderComposed::drawBar(QPainter* p) {
```

Comment:
```suggestion
void WSliderComposed::drawBar(QPainter* pPainter) {
```
And everywhere else in this function and the header file.

Reply:
done.